blueoil/blocks.py

```diff
@@ -169,15 +156,15 @@ def conv_bn_act(
         raise ValueError("data format must be 'NCHW' or 'NHWC'. got {}.".format(data_format))
 
     with tf.compat.v1.variable_scope(name):
-        conved = tf.layers.conv2d(
+        conved = tf.compat.v1.layers.conv2d(
             inputs,
             filters=filters,
             kernel_size=kernel_size,
             padding='SAME',
             use_bias=False,
-            kernel_initializer=tf.contrib.layers.variance_scaling_initializer(),  # he initializer
+            kernel_initializer=tf.compat.v1.keras.initializers.VarianceScaling(scale=2.0),  # he initializer
```

Comment:
`variance_scaling_initializer ` is same as `keras.initializers.VarianceScaling(scale=2.0)` ?